ManatiCR/manati-vm
# Try to add windows provisioning.

Fixes #38 

How to test:
- From a windows machine, clone the repo, change to this branch
- `vagrant up`
- Provisioning should start and work as expected
